readme.txt

```diff
@@ -284,7 +284,7 @@ If there currently isn't an active session, then calling this method does nothin
 COMMON FEATURES
 ========================================================================================================================================================
 
-User tracking
+Customers

```

Comment:
There is still a lot of mention of the word "user" in this Customers section compared to the .md file. Might have been missed?

Reply:
Haved changed this out. Lmk how this looks now :)